lib/rules/color-hex-case/__tests__/index.js

```diff
@@ -165,180 +165,180 @@ testRule(
 	}),
 );
 
-testRule({
-	ruleName,
-	config: ['lower'],
-	syntax: 'css-in-js',
-	fix: true,
-	accept: [
-		{
-			code: `
-				export const s = styled.a({
-					color: "#aaa",
-				});
-			`,
-		},
-		{
-			code: `
-				export const s = styled.a({
-					stroke: "url(#AAA)",
-				});
-			`,
-			description: 'href with location',
-		},
-		{
-			code: `
-				export const s = styled.a({
-					color: "PINK",
-				});
-			`,
-		},
-		{
-			code: `
-				export const s = styled.a({
-					background: "linear-gradient(#aaa, #ffff, #0000ffcc)",
-				});
-			`,
-		},
-		{
-			code: `
-				export const s = styled("a::before")({
-					content: '"#ABABAB"',
-				});
-			`,
-		},
-		{
-			code: `
-				export const s = styled.a({
-					color: "white /* #FFF */",
-				});
-			`,
-		},
-	],
+// testRule({
```

Comment:
[question] What is the reason for this comment-out? It might be good to leave the reason as a code comment.

E.g.

```suggestion
// NOTE: (describing comment-out reason...)
// testRule({
```

Reply:
Also, I prefer `/* ... */` over `// ...` because of fewer diffs, like other files.

Reply:
> Also, I prefer `/* ... */` over `// ...` because of fewer diffs, like other files.

Sorry, I understand using `// ...`. 🙇🏼  
Because `/* ... */` is unavailable for tests including `/* ... */` content.

Reply:
I've added a feature to skip specific `testRule` via stylelint/jest-preset-stylelint#33.

I can confirm it works on my localhost as below:

```console
npm i -D git+https://github.com/stylelint/jest-preset-stylelint.git#add-test-group-only-and-skip
```

```js
testRule({
	skip: true, //<-- add
	ruleName,
    // ...
});
```

Reply:
Thanks for adding this. It's much neater!

I've pushed up a commit to use `skip`.